data/avro-schema/src/main/resources/avro/net/corda/data/flow/event/FlowEvent.avsc

```diff
@@ -14,7 +14,8 @@
         "net.corda.data.flow.event.Wakeup",
         "net.corda.data.flow.output.FlowStatus",
         "net.corda.data.flow.event.SessionEvent",
-        "net.corda.data.persistence.EntityResponse"
+        "net.corda.data.persistence.EntityResponse",
+        "net.corda.data.crypto.wire.ops.flow.FlowOpsResponse"
```

Comment:
I'm sure it's on your radar already but this is the sort of thing where I'm hoping the unification work can help us out, by reducing the need to add something here for every subsystem we need to call out to.